run.py

```diff
@@ -174,6 +174,17 @@ def main(args):
   from nab.detectors.skyline.skyline_detector import SkylineDetector
   from nab.detectors.random.random_detector import RandomDetector
   from nab.detectors.null.null_detector import NullDetector
+  detectors = []
+  for detector in args.detectors:
+      if ':' in detector:
+        package, name = detector.split(':', 2)
+        className = detectorNameToClass(name)
+        imprt = __import__(package, fromlist=[className])
```

Comment:
It's [recommended to use importlib.import_module() over __import__()](docs.python.org/2/library/functions.html#__import__). So this would become 
```
from importlib import import_module  # at top of the file
...
import_module(package, className)
```

And please change the the variable name `imprt` to something more clear, like `moduleName`.